src/Microsoft.DotNet.Build.Tasks/Targets/packages.targets

```diff
@@ -61,7 +61,7 @@
 
     <GetPackageVersion
       Condition="Exists('$(ToolsDir)Microsoft.DotNet.Build.Tasks.dll')"
-      DailyBuildNumber="$(DailyBuildNumber)"
+      BuildNumber="$(BuildNumber)"
```

Comment:
What about repos that don't have a BuildNumber.props file setup? Is it going to be a new requirement that anyone using buildtools will need that file? If so that may be OK just want to be explicit about it. I guess they don't actually need the props file they just need to define that value itself.